js/network.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+window.network = (function () {
+
+  return {
+    errorHandler: function (message) {
+      console.log('Server error: ', message);
+    },
+
+    successHandler: function (message) {
+      console.log('Server responce: ', message);
+    },
+
+    receive: function (onLoad) {
```

Comment:
onError handler should be passed to this function as an argument.

Reply:
done